code/modules/mob/living/carbon/xenomorph/strains/castes/runner/acid.dm

```diff
@@ -83,7 +85,8 @@
 		modify_acid(acid_slash_regen_standing)
 
 /datum/behavior_delegate/runner_acider/on_life()
-	modify_acid(acid_passive_regen)
+	if(acid_amount <= acid_gen_cap)
```

Comment:
```suggestion
	if(acid_amount < acid_gen_cap)
```
Really ought to be `<` not `<=` else you will go above your cap whatever amount `acid_passive_regen` is (1)

Reply:
Oh rigth. Equal or less do be meaning that.